Porting/README.pod

```diff
@@ -381,6 +383,13 @@ to work on any of these, but it's a good idea to first contact
 I<perl5-porters@perl.org> to avoid duplication of effort, and to learn from
 any previous attempts.
 
+=head2 F<updateAUTHORS.pl>
+
+This script will automatically update AUTHORS and create .mailmap entries
+based on the git commit log history. If F<checkAUTHORS.pl> complains
```

Comment:
as stated here this cannot work outside a `git repo` no?

Reply:
Yes. Same as checkAUTHORS.pl itself. My assumption is that checkAUTHORS.pl doesn't run if its not in a git repo, so it wouldn't complain and people wouldn't be directed to use updateAUTHORS.pl :-)